osquery/sql/virtual_table.cpp

```diff
@@ -692,11 +692,13 @@ static int xBestIndex(sqlite3_vtab* tab, sqlite3_index_info* pIdxInfo) {
   // Expect this index to correspond with argv within xFilter.
   size_t expr_index = 0;
   // If any constraints are unusable increment the cost of the index.
-  double cost = 1;
+  double cost = 1000000;
 
   // Tables may have requirements or use indexes.
-  bool required_satisfied = false;
-  bool index_used = false;
+  int numIndexedColumns = 0;
```

Comment:
Is this used?

Reply:
Good call, was left after initial scoping work. Will remove.